tests/foreman/cli/test_contentviews.py

```diff
@@ -142,6 +142,46 @@ def setUp(self):
                 {u'organization-id': TestContentView.org['id']},
                 cached=True)
 
+    def test_bz1184442(self):
+        """@Test: Delete content view and verify it was actually deleted from
+        hard drive.
+
+        @Feature: Content View
+
+        @Assert: Content view was deleted and pulp folder doesn't contain
+        content view files anymore
+
+        @BZ: 1184442
+
+        """
+        # Create a content view and publish it
+        content_view = make_content_view({'organization-id': self.org['id']})
+        ContentView.publish({u'id': content_view['id']})
+        content_view = ContentView.info({u'id': content_view['id']})
+        # Check content view files presence before deletion
+        result = ssh.command(
+            'find /var/lib/pulp -name "*{0}*"'.format(content_view['name']))
+        self.assertEqual(result.return_code, 0)
+        self.assertNotEqual(len(result.stdout), 0)
+        # Completely delete the content view
+        self.assertEqual(len(content_view['versions']), 1)
+        ContentView.remove_from_environment({
+            'id': content_view['id'],
+            'lifecycle-environment-id':
+                content_view['lifecycle-environments'][0]['id'],
+        })
+        ContentView.version_delete({
+            'content-view': content_view['name'],
+            'organization': self.org['name'],
+            'version': content_view['versions'][0]['version'],
+        })
+        ContentView.delete({'id': content_view['id']})
+        # Check content view files presence after deletion
+        result = ssh.command(
+            'find /var/lib/pulp -name "*{0}*"'.format(content_view['name']))
+        self.assertEqual(result.return_code, 0)
+        self.assertEqual(len(result.stdout), 0)
```

Comment:
@sthirugn they are not the same. First batch of assertions checks that:
* Return code is '0'
* Something got returned (len > 0)
* Version is 1

then,

* Return code is '0'
* Nothing is returned (len == 0)